020-LinuxFundamentals/Student/Challenge-01.md

```diff
@@ -0,0 +1,26 @@
+# Challenge 01 - Create a Linux Virtual Machine
+
+**[Home](../README.md)** - [Next Challenge >](./Challenge-02.md)
+
+## Pre-requisites 
+
+- An active Azure Subscription.
+
+## Description
+
+To follow along you will need access to a server running a Linux-based operating system. Note that this fasthack was validated using a Linux server running Ubuntu 20.04 ([LTS](https://ubuntu.com/about/release-cycle)), but the examples given should work on any Linux distribution. See below the steps to follow to create your Linux Virtual Machine on Azure and choose that one with which you are more familiar.
+
+During the process of creation of the VM, ensure the usage of **student** for the username with administrator rights over the virtual machine, just to make it easier during the challenges. 
```

Comment:
Consider changing "administrator rights" to "root privileges" since "root privileges" is a Linux concept